pom.xml

```diff
@@ -89,7 +89,7 @@
     <logback.version>1.2.3</logback.version>
     <maven.project.version>2.2.1</maven.project.version>
     <mockito.version>3.9.0</mockito.version>
-    <sonar.version>7.9</sonar.version>
+    <sonar.version>8.8.0.42792</sonar.version>
```

Comment:
I think we could use 8.9 version, i.e. 8.9.0.43852. Also, can you also update `sonarQubeMinVersion`?